### DISCUSSION TITLE
[Feature] Middleware log monitor

### DISCUSSION THREAD

**Discussion by crazychengmm** [apache/skywalking#11840] [Q&A] [answered]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

We use Kafka reporter plugin to collect app log to Kafka，Use the web console to check the log ，Skywalking is the log solution we use. 
We have some middleware(es，kafka，minio) on k8s，is this any solution to collect the log to Skywalking？Then we can check all log on  Skywalking.
-- middleware log such as console log or file log.

### Use case

I want collect the middleware log to skywalking system.

### Related issues

no

### Are you willing to submit a pull request to implement this on your own?

- [X] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment, marked as answer]
We have provided log collectong for a while, check docs

- File logs, https://skywalking.apache.org/docs/main/next/en/setup/backend/filelog-native/
- OTEL logs, https://skywalking.apache.org/docs/main/next/en/setup/backend/log-otlp/
- Log Analysis, https://skywalking.apache.org/docs/main/next/en/setup/backend/log-analyzer/ , especially learn more on Log Analysis Language, LAL in SkyWalking concept

Meanwhile, we have provided log collecting from servers, like slow SQLs in MySQL, https://skywalking.apache.org/docs/main/next/en/setup/backend/backend-mysql-monitoring/#slow-sql-monitoring
